package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@codecademy/40phaser",
```

Comment:
Could you please `git checkout master -- package-lock.json`? We should resolve this separately. 

The change in name from a previous value to what it is now in `package.json` makes me think the changes here are correct, just out of place.